ts/screens/wallet/TransactionDetailsScreen.tsx

```diff
@@ -0,0 +1,154 @@
+/**
+ * Transaction details screen, displaying
+ * a list of information available about a
+ * specific transaction.
+ */
+import * as React from "react";
+
+import { Button, Content, Left, Right, Text, View } from "native-base";
+import { Image } from "react-native";
+import { Grid, Row } from "react-native-easy-grid";
+import {
+  NavigationInjectedProps,
+  NavigationScreenProp,
+  NavigationState
+} from "react-navigation";
+
+import { WalletStyles } from "../../components/styles/wallet";
+import { topContentTouchable } from "../../components/wallet/layout/types";
+import { WalletLayout } from "../../components/wallet/layout/WalletLayout";
+import I18n from "../../i18n";
+import { WalletTransaction } from "../../types/wallet";
+
+const cardsImage = require("../../../img/wallet/single-tab.png");
+
+interface ParamType {
+  readonly transaction: WalletTransaction;
+}
+
+interface StateParams extends NavigationState {
+  readonly params: ParamType;
+}
+
+type OwnProps = Readonly<{
+  navigation: NavigationScreenProp<StateParams>;
+}>;
+
+type Props = OwnProps & NavigationInjectedProps;
+
+/**
+ * Details of transaction
+ */
+export class TransactionDetailsScreen extends React.Component<Props, never> {
+  constructor(props: Props) {
+    super(props);
+  }
+
+  private touchableContent(): React.ReactElement<any> {
+    // TODO: replace this with actual component @https://www.pivotaltracker.com/story/show/157422715
+    return (
+      <View style={WalletStyles.container}>
+        <Image
+          style={WalletStyles.pfSingle}
+          source={cardsImage}
+          resizeMode="contain"
+        />
+      </View>
+    );
+  }
+
+  public render(): React.ReactNode {
+    const { navigate } = this.props.navigation;
+    const operation: WalletTransaction = this.props.navigation.state.params
+      .transaction;
+    const topContent = topContentTouchable(this.touchableContent());
+    return (
+      <WalletLayout
+        headerTitle={I18n.t("wallet.transaction")}
+        allowGoBack={true}
+        navigation={this.props.navigation}
+        title={I18n.t("wallet.transactionDetails")}
+        topContent={topContent}
+      >
+        <Content style={WalletStyles.whiteContent}>
+          <Grid>
+            <Row>
+              <Left>
+                <Text>{`${I18n.t("wallet.total")} ${operation.currency}`}</Text>
+              </Left>
+              <Right>
+                <Text style={WalletStyles.boldStyle}>{operation.amount}</Text>
```

Comment:
is this a different "bold" then the rest of the app?

Reply:
It isn't, but iinm there isn't a "bold" definition in `Text`. Can you have a look at my [latest commit](https://github.com/teamdigitale/italia-app/pull/162/commits/f7d740d8beae97156dafee5cf59b95e5d05058f0) to see whether that's more like what you were expecting?  

Other styles in `ts/components/styles/wallet.ts` are setting `fontWeight` independently, but I'm planning on fixing that as part of [157769361](https://www.pivotaltracker.com/story/show/157769361)

Reply:
@cloudify any feedback about this?

Reply:
@fpersico